account.go

```diff
@@ -137,7 +137,7 @@ func (account *Account) fmtExecute(ctx context.Context, calls []types.Transactio
 	req := types.FunctionInvoke{
 		FunctionCall: types.FunctionCall{
 			ContractAddress:    account.Address,
-			EntryPointSelector: EXECUTE_SELECTOR,
+			EntryPointSelector: types.StrToFelt(EXECUTE_SELECTOR),
```

Comment:
It does not work that way... When dealing with entrypoints named `function`, starknet uses `keccak256(function) & (250bits)` as you can see from `GetSelectorFromName`. You cannot take a string a bind it to a Felt like that. What I suggess for now is you leave `EntryPointSelector` in `types FunctionCall` a string. Even though I am pretty sure it might contain the `GetSelectorFromName` of the function